src/physics/base/PhysicsTrackView.i.hh

```diff
@@ -271,13 +360,25 @@ CELER_FUNCTION real_type PhysicsTrackView::linear_loss_limit() const
     return params_.linear_loss_limit;
 }
 
+//---------------------------------------------------------------------------//
+/*!
+ * Energy scaling fraction used to estimate maximum cross section over a step.
+ *
+ * This parameter is defined as \f$ \xi = 1 - \alpha \f$, where \f$ \alpha \f$
+ * is \c scaling_fraction.
+ */
+CELER_FUNCTION real_type PhysicsTrackView::energy_fraction() const
+{
+    CELER_EXPECT(params_.scaling_fraction <= 1);
+    return 1 - params_.scaling_fraction;
```

Comment:
If this is a settable parameter, use parms_.scaling_fraction instead of 1 -  parms_.scaling_fraction ?

Reply:
I know `energy_fraction` (or `lambdaFactor` in Geant4 parlance) is by default `1 - dRoverRange` (which we call `scaling_fraction`) -- is it also a user-configurable parameter? Maybe I should store `energy_fraction` in the `PhysicsParamsData` and add it as an input option?